examples/darwin-framework-tool/commands/common/DFTKeypair.mm

```diff
@@ -0,0 +1,239 @@
+/*
+ *   Copyright (c) 2024 Project CHIP Authors
+ *   All rights reserved.
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ *
+ */
+
+#import "DFTKeypair.h"
+
+#import "CHIPCommandStorageDelegate.h"
+#import "ControllerStorage.h"
+
+#define CAKeyTag "com.apple.matter.commissioner.ca.issuer.id"
+#define KeySize "256"
+#define CHIP_CRYPTO_SYMMETRIC_KEY_LENGTH_BYTES 16
+
+static NSString * const kKeychainLabel = @"Chip Tool Keypair";
+static NSString * const kOperationalCredentialsIssuerKeypairStorage = @"ChipToolOpCredsCAKey";
+static NSString * const kOperationalCredentialsIPK = @"ChipToolOpCredsIPK";
```

Comment:
I assume these are "ChipTool" and not "DarwinFrameworkTool" for compat with existing storages?  If so, worth documenting that.